packages/sil-voting-app/src/routes/dashboard/index.tsx

```diff
@@ -1,16 +1,18 @@
 import Block from 'medulas-react-components/lib/components/Block';
 import Hairline from 'medulas-react-components/lib/components/Hairline';
-import Typography from 'medulas-react-components/lib/components/Typography';
 import React from 'react';
 import Header from '../../components/Header';
+import AsideFilter from './components/AsideFilter';
+import ProposalsList from './components/ProposalsList';
 
 const Dashboard = (): JSX.Element => {
   return (
     <Block width="100%" maxWidth="1024px" height="auto" display="flex" flexDirection="column" margin="0 auto">
```

Comment:
Why don't you want to use `Grid` component instead of `Block` in this case?

Reply:
`Grid` doesn't have a `margin` property so I can't horizontal align it.

Reply:
Yes but you can use nested component for this. But I thought a little about it and think `Grid` component idea is not suitable in this case. Sorry for confusing.